src/org/omegat/Bundle.properties

```diff
@@ -2036,10 +2036,16 @@ GUI_LANGUAGETOOL_URL=URL:
 
 GUI_LANGUAGETOOL_RULES=Rules
 
-GUI_LANGUAGETOOL_BAD_RULE_ID=Rule ID can only contain latin letters and underscores.
+GUI_LANGUAGETOOL_BAD_RULE_ID=Rule ID can only contain Latin letters and underscores.
 
```

Comment:
During debugging network bridge I found out that there are rules with dots in names, so I changed regexp to `"^[A-Za-z_\\.]+$"` Not sure if LT devs just overlooked those rule names in their code.